examples/AspNetCore/Program.cs

```diff
@@ -26,20 +26,20 @@
 var appBuilder = WebApplication.CreateBuilder(args);
 
 // Note: Switch between Zipkin/Jaeger/OTLP/Console by setting UseTracingExporter in appsettings.json.
-var tracingExporter = appBuilder.Configuration.GetValue<string>("UseTracingExporter").ToLowerInvariant();
+var tracingExporter = appBuilder.Configuration.GetValue<string>("UseTracingExporter")!.ToLowerInvariant(); // "UseTracingExporter" is defined in appSettings.json
```

Comment:
nit: It might make for a better example if we had a default specified in these lookups just in case the settings aren't found? For example:

```csharp
var tracingExporter = appBuilder.Configuration.GetValue("UseTracingExporter", "console").ToLowerInvariant();
```

With the default you wouldn't need the null suppressing `!` sprinkled around either.

Reply:
I provided the default values, but it still complains about possible null reference.

Reply:
Looks like the API has the nullable return annotation even when the default is passed:

`public static T? GetValue<T> (this Microsoft.Extensions.Configuration.IConfiguration configuration, string key, T defaultValue);`

That's lame sorry 😄